modules/logging-es-storage-considerations.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-deploying.adoc
+
+[id="logging-es-storage-considerations_{context}"]
+= Storage considerations for Elasticsearch
+
+A persistent volume is required for each Elasticsearch deployment configuration. On {product-title} this is achieved using persistent volume claims.
```

Comment:
Since you use `PVC` later, it is worth defining the acronym here.
```suggestion
A persistent volume is required for each Elasticsearch deployment configuration. On {product-title} this is achieved using persistent volume claims (PVCs).
```